docs/Installing.asciidoc

```diff
@@ -411,10 +411,12 @@ connecting to a remote PostgreSQL database with a username and password:
 [source, ini]
 --------------------------------------------------------------------------------
 [production]
-dsn = dbi:Pg:dbname=openqa;host=db.example.org
+dsn = dbi:Pg:dbname=openqa
```

Comment:
so better make it two examples:
* update the description for this example to be local, not remote
* add a subsection with an advanced example for remote connection